.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.17.0
```

Comment:
Will this still work with v14 we currently use?

Reply:
I am not 100% sure